scripts/stablex/add_token.js

```diff
@@ -1,3 +1,4 @@
+const { addTokens } = require("../script_utilities.js")
```

Comment:
actually, the addTokens method should probably live in a utilities file inside the snapp subfolder since it doesn't apply to dfusion.

Reply:
done